app/views/layouts/application.html.erb

```diff
@@ -3,12 +3,35 @@
   <head>
     <title>CfpXXIV</title>
     <%= csrf_meta_tags %>
-
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Lato|Lora" rel="stylesheet">
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
-    <%= yield %>
+    <header class="header">
+      <h1>Call 4 Papers</h1>
+    </header>
+    <section class="about">
+      <h2>Sobre a Semana da Tecnlogia</h2>
+      <p>A Semana da Tecnologia é um evento anual da Faculdade de Tecnologia de Sorocaba(FATEC), nele são
+        apresentados palestras e ministrados cursos sobre os assuntos de maior destaque no mercado. Este ano
+        será realizada a 24º edição do evento, sendo que será a segunda edição que está sendo organizada pelos
+        alunos do curso de Análise e Desenvolvimento de Sistemas.
+      </p>
+      <p>
+        Para saber mais entre no <a href="#"><i>site oficial do evento</i></a>
```

Comment:
here you can change the `#` to the real website https://fatecsorocaba.github.io/semana-da-tecnologia/